readme.md

```diff
@@ -40,6 +40,32 @@ console.log(obj);
 //=> {foo: {bar: 'b', baz: 'x', 'dot.dot': 'unicorn'}}
 ```
 
+
+## API
+
+### get(obj, path)
+
+### set(obj, path, value)
+
+#### obj
+
+Type: `object`
+
+Object to get or set the path value.
```

Comment:
I'd say something like:
Object that the `get` or `set` operation work on

My English is bad though :disappointed: